src/neuroconv/tools/data_transfers/_aws.py

```diff
@@ -30,3 +36,289 @@ def estimate_s3_conversion_cost(
     cost_gb_m = 0.08 / 1e3  # $0.08 / GB Month
     cost_mb_s = cost_gb_m / (1e3 * 2.628e6)  # assuming 30 day month; unsure how amazon weights shorter months?
     return cost_mb_s * total_mb_s
+
+
+def submit_aws_batch_job(
+    *,
+    job_name: str,
+    docker_image: str,
+    command: Optional[str] = None,
+    job_dependencies: Optional[List[Dict[str, str]]] = None,
+    region: str = "us-east-2",
+    status_tracker_table_name: str = "neuroconv_batch_status_tracker",
+    iam_role_name: str = "neuroconv_batch_role",
+    compute_environment_name: str = "neuroconv_batch_environment",
+    job_queue_name: str = "neuroconv_batch_queue",
+    job_definition_name: Optional[str] = None,
+    minimum_worker_ram_in_gb: float = 4.0,
```

Comment:
```suggestion
    minimum_worker_ram_in_gib: int = 4,
```

Maybe you're right though, since it does determine all kinds of allocation stuff in the cloud